basis_universal.lua

```diff
@@ -19,6 +19,10 @@ flags {
   "NoPCH"
 }
 
+excludes {
+  "basisu_tool.cpp"
```

Comment:
Instead of this can we remove line 31 - 33

```
"*.h",
  "*.cpp",
  "*.c",
```

and delete the files we don't need

Reply:
> Instead of this can we remove line 31 - 33
> 
> ```
> "*.h",
>   "*.cpp",
>   "*.c",
> ```
> 
> and delete the files we don't need

If I delete lines 31-33, wouldn't that cause .h, .c and .cpp files not to be part of the build?

Reply:
I was looking at the latest version os basis which has a different folder layout. We should look at going to the latest version